cli/data.py

```diff
@@ -79,10 +81,16 @@ def update(summary_filename, wide_dates_filename):
     timeseries_dataset: TimeseriesDataset = combined_datasets.build_from_sources(
         TimeseriesDataset, data_sources, ALL_TIMESERIES_FEATURE_DEFINITION, filter=US_STATES_FILTER
     )
-    multiregion_dataset = MultiRegionTimeseriesDataset.from_timeseries(timeseries_dataset)
     latest_dataset: LatestValuesDataset = combined_datasets.build_from_sources(
         LatestValuesDataset, data_sources, ALL_FIELDS_FEATURE_DEFINITION, filter=US_STATES_FILTER,
     )
+    multiregion_dataset = MultiRegionTimeseriesDataset.from_timeseries_and_latest(
+        timeseries_dataset, latest_dataset
+    )
+    if aggregate_to_msas:
+        aggregator = statistical_areas.CountyToCBSAAggregator.from_local_public_data()
+        multiregion_dataset = aggregator.aggregate(multiregion_dataset)
```

Comment:
I might have mentioned this before, but thoughts on returning a new data frame instead of the combined dataset? Gives us more flexibility with ohow we use the aggregated data

Reply:
I made the change you suggested before my last merge to master but didn't update this. Done now with test for merge too.